types/src/transaction.rs

```diff
@@ -401,8 +401,8 @@ impl SignedTransaction {
             raw_txn: raw_txn.clone(),
             public_key,
             signature,
-            // In real world raw_txn should be derived from raw_txn_bytes, not the opposite.
-            raw_txn_bytes: raw_txn.into_proto_bytes().expect("Should convert."),
+            raw_txn_bytes: SimpleSerializer::<Vec<u8>>::serialize(&raw_txn)
```

Comment:
Annoyingly, I can't seem to comment on unchanged lines, so I'll add the comment here.  Why do we still have raw_txn_bytes in SignedTransaction?  It seems like we would only really need it in case we wanted to be able to provide the client with an error message like "Incorrectly serialized"

Reply:
Agreed. This will require non-trivial fixes to the core of Libra. I didn't want to incur the wrath of making yet another massive PR. So I'll hopefully have that ready Monday afternoon or Tuesday. So we can land it shortly after the next cut.

Reply:
Actually... maybe we can break it up into two things... one is to fix the proto / grpc call. The second would be to eliminate this field on the rust data structure -- that way we can easily make the internal change without impact clients.